simulator/http-simulator.php

```diff
@@ -2,18 +2,25 @@
 require_once __DIR__ . '/../vendor/autoload.php';
 $uri = $_SERVER['REQUEST_URI'];
 switch (true) {
-    case preg_match('/sitemap\.xml/', $uri, $matches):
-        echo file_get_contents(__DIR__ . "/responses/sitemap.xml");
+    case preg_match('~gb/en/sitemap\.xml~', $uri, $matches):
+        echo file_get_contents(__DIR__ . "/responses/britishSiteMap.xml");
+        break;
+    case preg_match('~nl/nl/sitemap\.xml~', $uri, $matches):
+        echo file_get_contents(__DIR__ . "/responses/dutchSiteMap.xml");
+        break;
+    case preg_match('/sitemap([0-9]?)\.xml/', $uri, $matches):
+        $siteMapNumber = is_numeric($matches[1]) ? $matches[1] : '';
+        echo file_get_contents(__DIR__ . "/responses/sitemap{$siteMapNumber}.xml");
         break;
     case preg_match('/timeout/', $uri, $matches):
```

Comment:
could we do something like endsWith (what's the equivalent in php again ?) instead of using preg_match ? This is going to redirect all urls with "timeout" and "alive" in them in the wrong place isn't it?